tests/integration/istioctl/webhook/webhook_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istioctl
+
+import (
+	"regexp"
+	"strings"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework/label"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/istioctl"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/framework/resource"
+)
+
+var (
+	inst istio.Instance
+	g    galley.Instance
+	p    pilot.Instance
+)
+
+// This test requires `--istio.test.env=kube` because it tests istioctl managing k8s webhook configurations.
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite("istioctl_webhook_test", m).
+		Label(label.CustomSetup).
+		RequireEnvironment(environment.Kube).
+		// Deploy Istio
+		SetupOnEnv(environment.Kube, istio.Setup(&inst, setupConfig)).
+		Setup(func(ctx resource.Context) (err error) {
+			if g, err = galley.New(ctx, galley.Config{}); err != nil {
+				return err
+			}
+			if p, err = pilot.New(ctx, pilot.Config{
+				Galley: g,
+			}); err != nil {
+				return err
+			}
+			return nil
+		}).
+		Run()
+}
+
+func setupConfig(cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+	// Helm values from install/kubernetes/helm/istio/test-values/values-istio-dns-cert.yaml
+	cfg.ValuesFile = "test-values/values-istio-dns-cert.yaml"
+	cfg.Values["global.operatorManageWebhooks"] = "true"
+}
+
+// TestWebhookManagement tests "istioctl experimental post-install webhook" command.
+func TestWebhookManagement(t *testing.T) {
+	framework.
+		NewTest(t).
+		Run(func(ctx framework.TestContext) {
+			g := galley.NewOrFail(t, ctx, galley.Config{})
```

Comment:
You're already doing this in your main ... you don't need this here.

Are you even using these variables? If not you don't need this logic here or above.

Reply:
Removed.